editor/editor_settings.cpp

```diff
@@ -1247,7 +1239,7 @@ String EditorSettings::get_settings_dir() const {
 }
 
 String EditorSettings::get_project_settings_dir() const {
-	return get_settings_dir().plus_file("projects").plus_file(project_config_dir);
+	return EditorSettings::PROJECT_EDITOR_SETTINGS_PATH;
```

Comment:
Since there's a getter for it specifically, maybe it doesn't need to be a separate constant? You could just use `get_project_settings_dir()` in `create()`.

Reply:
The original idea was to remove the methods and only have the constant, but the method is still needed if we want to expose this (we can't bind string constants AFAIK, just numeric ones).